### PR TITLE
Add github commit status with version

### PR DESCRIPTION
**Pull request by tomverin**

### CODE REVIEW

**Comment by tomverin** [comment]
image
Looks like this on sonar-dummy latest commit

File: bin/cirrus_promote_maven [hunk 1]
```diff
@@ -7,9 +7,14 @@ set -euo pipefail
 source cirrus-env PROMOTE
 source set_maven_build_version $BUILD_NUMBER
 curl -sfSL -H "Authorization: Bearer $GCF_ACCESS_TOKEN" "$PROMOTE_URL/$GITHUB_REPO/$GITHUB_BRANCH/$BUILD_NUMBER/$PULL_REQUEST"
+
+#Add github commit status with build number
+BUILD_URL="${ARTIFACTORY_URL}/webapp/#/builds/sonar-dummy/${BUILD_NUMBER}/"
```
Comment (malena-ebert-sonarsource) [inline bin/cirrus_promote_maven:12]: `sonar-dummy` should be replaced with the actual project name.

**Review by malena-ebert-sonarsource — CHANGES_REQUESTED** [review]

File: bin/cirrus_promote_maven (hunk 1 above)
Comment (tomverin) [reply bin/cirrus_promote_maven:12]: oups indeed, thanks

**Review by malena-ebert-sonarsource — APPROVED** [review]
LGFM

**Pull request merged**